hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerProtocolServerSideTranslatorPB.java

```diff
@@ -487,7 +487,13 @@ private OMRequest associateBucketIdWithRequest(OMRequest omRequest)
       return omRequest;
     }
 
-    long bucketId = metadataManager.getBucketId(volumeName, bucketName);
+    long bucketId;
+    try {
+      bucketId = metadataManager.getBucketId(volumeName, bucketName);
+    } catch (OMException oe) {
+      // Ignore exceptions at this stage, let respective classes handle them.
+      return omRequest;
+    }
```

Comment:
Hi @rakeshadr, could you please take a look at this added logic inside the new method?